samples/display/st7789v/README.rst

```diff
@@ -0,0 +1,41 @@
+.. _st7789v-sample:
+
+ST7789V Display driver
+######################
+
+Overview
+********
+This sample will draw some basic rectangles onto the display.
```

Comment:
Should make a reference to the hardware, assuming this is the right link:
```
This sample draws rectangles onto an `ST7789V display
<https://www.crystalfontz.com/controllers/Sitronix/ST7789V/>.
```
Or is there a link to the specific display module you tested this with?

Reply:
I added a couple of links, but as I say in the comments, the specific module that I dev/tested this with was a custom board.